Foundation/NSArray.swift

```diff
@@ -84,7 +84,7 @@ public class NSArray : NSObject, NSCopying, NSMutableCopying, NSSecureCoding, NS
     }
     
     public func copyWithZone(zone: NSZone) -> AnyObject {
-        return self
+        return NSArray(array: self.allObjects)
```

Comment:
NSArray is immutable so it just should return self, subclasses need to do the O(n) copy